packages/xrpc-stream/src/stream.ts

```diff
@@ -0,0 +1,16 @@
+import { DuplexOptions } from 'stream'
+import { createWebSocketStream, WebSocket } from 'ws'
+import { Frame } from './frames'
+
+export async function* createFrameStream(
```

Comment:
Streams are async iterables, but async iterables aren't necessarily streams, so I may have misnamed this a bit.  It's hard to name, maybe something like `byFrame(ws)`?  🤔